app/javascript/components/forms/DownloadCsv.vue

```diff
@@ -43,8 +43,7 @@
 
         axios.post('/pame/download', data, config)
           .then((response) => {
-            const date = new Date().toJSON().slice(0,10),
-              filename = `protectedplanet-pame-${date}.csv`
+            const filename = response.headers['content-disposition'].split('filename\="')[1].split('\"')[0]
```

Comment:
Seems like this could benefit from more clarity with regards to what [1] and [0] are? I'm guessing `[1]` is the individual header value for the filename including the path and `[0]` is the filename itself?

Reply:
You are right, I've just added a comment that explains that. Thanks!